sway/commands.c

```diff
@@ -1124,9 +1124,26 @@ static struct cmd_results *cmd_bar(int argc, char **argv) {
 	bar->tray_padding = config->bar.tray_padding;
 	list_add(config->bars, bar);
 
+	// set bar id
+	int i, n;
+	int len = 5; // "bar-" + \0
```

Comment:
`numlen` is defined in `swaygrab/main.c`, where should I put it for general use?